src/js/modules/generateKeybord.js

```diff
@@ -0,0 +1,77 @@
+import keysData from './keysData';
+import createElement from './createElement';
+
+const { body } = document;
+const textarea = createElement('textarea', 'textarea');
+body.append(textarea);
+
+const keybord = createElement('div', 'keybord');
```

Comment:
If you are using VSCode editor try to download and install SpellChecker extension. It helps to avoid writting typos and mistakes.